src/applications/static-pages/authenticated-homepage/components/App/App.js

```diff
@@ -0,0 +1,53 @@
+// Node modules.
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { selectShowAuthenticatedHomepage } from 'applications/static-pages/authenticated-homepage/selectors';
+import environment from 'platform/utilities/environment';
+
+export const App = ({ showAuthHomePage }) => {
+  const homepage = document.getElementById('homepage');
+
+  if (environment.isProduction() || !showAuthHomePage || !homepage) {
+    return null;
+  }
+
+  homepage.style.display = 'none';
+  return <div>This is the new authenticated homepage.</div>;
+};
+
+App.propTypes = {
+  showAuthHomePage: PropTypes.bool,
+};
+
+const mapStateToProps = state => {
+  const loggedIn = state.user.login.currentlyLoggedIn;
+  const authHomepageFeatureFlag = selectShowAuthenticatedHomepage(state);
+  const authHomepageLocalStorage = localStorage.getItem(
+    'AUTHENTICATED_HOMEPAGE',
+  );
+  // This value exists to override the feature flag when it's on
+  const noAuthHomepage = localStorage.getItem('NO_AUTHENTICATED_HOMEPAGE');
```

Comment:
rather than using a second local storage value, why not treat `AUTHENTICATED_HOMEPAGE === 'false'` as the way to force the feature off?